packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -744,8 +744,8 @@ test.serial('writeGranulesFromMessage() on re-write saves granule records to Dyn
     granule_id: granuleId,
     collection_cumulus_id: collectionCumulusId,
   });
-  const apiFormattedPostgresGranule =
-    await translatePostgresGranuleToApiGranule({
+  const apiFormattedPostgresGranule
+    = await translatePostgresGranuleToApiGranule({
```

Comment:
Nit: we should consider fixing this opinionated linting one way or another- having the assignment on the next line is jarring (operator linebreak) is not part of our lint set, and prettier format defaults to 

```
  const apiFormattedPostgresGranule =
    await translatePostgresGranuleToApiGranule({

      granulePgRecord: postgresRecord,
      knexOrTransaction: knex,
    });
```

I'm fine making the argument we should do this, but we should do it everywhere.